modules/ding_provider/connie_search/src/ConnieSearchResult.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @file
```

Comment:
Is this a Drupal 8 convention? Scrutinizer has not run for this PR, but we have not seen this issue as a part of our own development process which have been running with the same configuration as Core.

Reply:
Don't really know if it's a change in the coding standards from 7 to 8. I just use php code sniffer with Drupal as the standard.

@kasperg  I'll let you decide (as I think it might be something new or at least something that we have not used before).